ansible/roles/accumulo/templates/generic_logger.xml

```diff
@@ -31,7 +35,11 @@
 
   <!--  Write out INFO and higher to the regular log -->
   <appender name="A3" class="org.apache.log4j.RollingFileAppender">
+     {% if accumulo_major_version == '1' %}
      <param name="File"           value="${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${org.apache.accumulo.core.ip.localhost.hostname}.log"/>
+     {% else %}
+     <param name="File"           value="${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${accumulo.service.instance}${org.apache.accumulo.core.ip.localhost.hostname}.log"/>
```

Comment:
`accumulo.service.instance` has no affect on 1.7 if added.  Therefore, I added it and removed the logic.